utils/const.js

```diff
@@ -29,6 +29,8 @@ module.exports = {
   UPDATE_SIGNAL_FILE: process.env.UPDATE_SIGNAL_FILE || '/signals/update',
   UPDATE_LOCK_FILE: process.env.UPDATE_LOCK_FILE || '/statuses/update-in-progress',
   BACKUP_STATUS_FILE: process.env.BACKUP_STATUS_FILE || '/statuses/backup-status.json',
+  TOR_PROXY_IP: process.env.TOR_PROXY_IP,
+  TOR_PROXY_PORT: process.env.TOR_PROXY_PORT,
```

Comment:
Maybe set some sensible defaults for these values?

I'd suggest 192.168.0.1 and 9050, the default when Tor is running on the same machine.

They should also be documented in the table in the readme: https://github.com/getumbrel/umbrel-manager#step-2-set-environment-variables